sample_app_dagger/src/main/java/com/chumarin/stanislav/sample_app_dagger/util/BaseViewModel.java

```diff
@@ -0,0 +1,9 @@
+package com.chumarin.stanislav.sample_app_dagger.util;
+
+import androidx.lifecycle.ViewModel;
+
+public class BaseViewModel extends ViewModel {
+
```

Comment:
Looks like missing implementation.

In the Koin sample app, this class tracks ViewModel instances, enabling us to verify expected lifecycle changes as session tracker state changes.

Reply:
Added similar implementation